pkg/tracker/tracker.go

```diff
@@ -51,7 +51,11 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
-	Tracker = &tracker{gaPropertyID: os.Getenv(GA_TRACKING_ID), cid: id.String()}
+	Tracker = &tracker{
+		gaPropertyID: os.Getenv(GA_TRACKING_ID),
+		cid:          id.String(),
+		gaAPIURL:     "https://www.google-analytics.com/collect",
```

Comment:
Please define env var `GA_TRACKING_URL`. If empty we should set default `https://www.google-analytics.com/collect`

Reply:
done